packages/components/organisms/f-cookie-banner/README.md

```diff
@@ -0,0 +1,87 @@
+
+<div align="center">
+  <h1>f-cookie-banner</h1>
+
+  <img width="125" alt="Fozzie Bear" src="../../bear.png" />
+
+  <p>Cookie Banner</p>
+</div>
+
+---
+
+[![npm version](https://badge.fury.io/js/%40justeat%2Ff-cookie-banner.svg)](https://badge.fury.io/js/%40justeat%2Ff-cookie-banner)
+[![CircleCI](https://circleci.com/gh/justeat/fozzie-components.svg?style=svg)](https://circleci.com/gh/justeat/workflows/fozzie-components)
+[![Coverage Status](https://coveralls.io/repos/github/justeat/f-cookie-banner/badge.svg)](https://coveralls.io/github/justeat/f-cookie-banner)
+[![Known Vulnerabilities](https://snyk.io/test/github/justeat/f-cookie-banner/badge.svg?targetFile=package.json)](https://snyk.io/test/github/justeat/f-cookie-banner?targetFile=package.json)
+
+
+## Usage
+
+1.  Install the module using NPM or Yarn:
+
+    ```bash
+    yarn add @justeat/f-cookie-banner
+    ```
+
+    ```bash
+    npm install @justeat/f-cookie-banner
+    ```
+
+2.  Import the component
+
+    You can import it in your Vue SFC like this (please note that styles have to be imported separately):
+
+    ```
+    import CookieBanner from '@justeat/f-cookie-banner';
+    import '@justeat/f-cookie-banner/dist/f-cookie-banner.css';
+
+    export default {
+        components: {
+            CookieBanner
+        }
+    }
+    ```
+
+    If you are using Webpack, you can import the component dynamically to separate the `cookie-banner` bundle from the main `bundle.client.js`:
+
+    ```
+    import '@justeat/f-cookie-banner/dist/f-cookie-banner.css';
+
+    export default {
+        components: {
+            ...
+            CookieBanner: () => import(/* webpackChunkName: "cookie-banner" */ '@justeat/f-cookie-banner')
+        }
+    }
+
+    ```
+
+## Configuration
+
+### Props
+
+`f-cookie-banner` has a number of props that allow you to customise its functionality.
+
+The props that can be defined are as follows:
+
+| Prop  | Type  | Default | Description |
+| ----- | ----- | ------- | ----------- |
+| `locale` | `String` | `'en-GB'` | Set the tenant for localisation ['da-DK', 'en-AU', 'en-GB', 'en-IE', 'en-NZ', 'es-ES', 'it-IT', 'nb-NO']. |
+| `isHidden` | `Boolean` | `false` | show/hide the cookie consent banner. |
+| `showLegacyBanner` | `Boolean` | `false` | Use the legacy "passive" banner markup. |
+| `cookieExpiry` | `Number` | `7776000` | Expiry time of cookies written to the browser. |
+
+## Development
+
+Running below `yarn` commands from the component folder, starts a development
+server displaying a preview example of the component implementation.
+
+```bash
+# cd /packages/f-cookie-banner
+yarn install
+
+# followed by
+yarn demo
```

Comment:
This is out of date I am afraid. You can copy the right instructions from [here](https://github.com/justeat/fozzie-components/tree/master/packages/components/atoms/f-button#development)

Reply:
Done